### DISCUSSION TITLE
Right Click Menus

### DISCUSSION THREAD

**Discussion by Alienmoon** [ramensoftware/windhawk#307] [Q&A] [answered]
Hi, I'm new to Windhawk, so forgive me if this has already been answered, but I couldn't find a solution within Windhawk, even using search within Discussions, I can't find anything under Right Click menus.

I like using Windhawk as it covers 90% of my needs. I enjoy the Translucent/Acrylic look, and I have these effects applied to my Notification area, Start menu, taskbar, and Explorer windows. However, I can't seem to find a mod within Windhawk that covers "Right-Click Menus." or am I missing something?

I've tried installing AcrylicMenus though I can't seem to get the effects to apply, so my question is, Is this MOD already in Windhawk under some weird name and if not, will this mod be coming to Windhawk anytime soon?

Thanks for all the awesome mods and the work you do to keep this app going.

**Comment by m417z** [comment, marked as answer]
Hi,

There are currently no Windhawk mods to style context menus. There are some mods related to menus ([example](https://windhawk.net/mods/explorer-context-menu-classic)), but it doesn't seem to be what you're looking for.

As for "will this mod be coming to Windhawk anytime soon", I don't have plans for it, but who knows, maybe somebody will create a mod for that at some point.

**Reply by Alienmoon** [reply]
Thanks for the reply mate and yeah, the classic look is not what I'm looking for, so I guess I'll keep my fingers crossed and hope someone adds a mod that'll give Translucent / Acrylic context menus in the future, as it'll complete the Translucent/Acrylic look all from one app, which I think would be awesome.

thanks again :)

**Reply by stamminator** [reply]
[Nilesoft Shell](https://github.com/moudey/Shell) does an excellent job of this. It's customizable, actually runs faster than the default context menus, and looks great. Just be aware of the downsides:
- The repo is pretty messy. The maintainer disappears for months at a time, packages aren't handled well, the documentation site often goes down and is painful to use, and there's no rhyme or reason to releases, with issues that were fixed several months ago still being broken in the latest GitHub-hosted release. [The "canary" version directly on the website](https://nilesoft.org/download) is stable for me, but this really shouldn't be necessary. It's approaching abandonware territory.
- No GUI configuration. Customizing it requires modifying `nss` files, although there is a VS Code extension which helps. Here's an example of a menu option I added which let's you right-click on a network drive and RDP into its host machine:
```
item(title='Open RDP Session' icon=\uE1ED type='dir' cmd args='/c StartRDP "@sel.lnk"' window=hidden )
```

<img width="350" alt="cZ4Cd4MYZA" src="https://github.com/user-attachments/assets/4e469e83-9174-4423-84cd-28edcb78530e">

____

All that being said, it's the best one out there. I would be in support of a Windhawk fork of the project to get it into a proper state. It could be called "Reshell" to differentiate it from the million other things in software called "shell"
